server/plugins/api/LiveSite.ts

```diff
@@ -0,0 +1,24 @@
+import { APIReturn } from "./APIReturn"
+import { GenericNotifier } from "./GenericNotifier"
+import { GenericConfig } from "./GenericConfig"
+
+interface Config extends GenericConfig {
+  groups?: string[],
```

Comment:
Let's not think about `groups` yet, we don't need them for MVP and they might be more complicated then I first thought.